### ISSUE TITLE
2.8.4 Patch Release Plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3101]
- #3098
- #2953
- #3038
- #3040
- #3066
- #3056
- #3072
- #3078

## Pick list
```
git cherry-pick 5450f98befa48bff8e3b97fd4868436b0d759694
git cherry-pick 7f2875c9851e1c64bdea6cc445b42608bccc7119
git cherry-pick b1af466b7e006b90f47ce066e351ca185f887ecd
git cherry-pick 7590d25d0105f484cb1b141131e0a97c3f09c854
git cherry-pick de2c2cf6529159f9970ff8f6929139a4d5b412a6
git cherry-pick 35d86c8974fb45a5c29895b6f9b73ad2a15dc218
git cherry-pick 5c387920f4633126a0df66c7e49a6dd953f423b0
```

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.8.3..HEAD --pretty='format:- %s'`
```
_Note: This release contains a security update for Drupal core that addresses critical security vulnerability ([SA-CORE-2024-001](https://www.drupal.org/SA-CORE-2024-001)).  Only sites that have the core comment module installed are affected by this vulnerability._

- Closes #3077: Address PHP 8.1+ deprecation warnings caused by card links with empty URLs. (#3078)
- Closes #3060 - Fix CKEditor editor experience to match previous version (#3072)
- Close #2800 Update Global Footer with new X/Twitter Icon. (#3056)
- Closes #3065: Remove text decoration from links in az_publication.module (#3066)
- #2084: Add DB update to uninstall now unsupported media_library_theme_reset module. (#3040)
- #3014: Add critical new block_content permissions to AZQS roles (added in Drupal 10.1.x) (#3038)
- Closes #2952 Clone permissions in the Quick Node Clone module do not reflect custom access rules (#2953)
- #3099 Update Drupal core to 10.1.8 for SA-CORE-2024-001 (2.8.x) (#3098)
- Prepare 2.8.x branch for new minor release branch 2.9.x (#3052)
- Back to dev.
```

**Comment by joeparsons** [comment]
https://github.com/az-digital/az_quickstart/releases/tag/2.8.4
